flask_resty/testing.py

```diff
@@ -28,11 +29,12 @@ def get_meta(response):
 
 def assert_value(actual, expected):
     if isinstance(expected, dict):
-        assert isinstance(actual, dict)
+        assert isinstance(actual, Mapping)
         for k, v in expected.items():
             assert_value(actual.get(k, None), v)
-    elif isinstance(expected, list):
-        assert isinstance(actual, list)
+    elif isinstance(expected, Sequence) \
```

Comment:
Also, I don't think you need to assert that `actual` is a `Sequence` – the `zip_longest()` will just fail if it isn't, right?